ops/k8s/deployments/api-deployment.yaml

```diff
@@ -101,3 +101,8 @@ spec:
             secretKeyRef:
               name: api-secrets
               key: stairway-password
+        - name: SPRING_PROFILES_ACTIVE
+          valueFrom:
+            secretKeyRef:
```

Comment:
does this info really need to be secret? are we just getting it from the secrets so we can parameterize it?

Reply:
yes I'd like to revisit and separate some of these into configs, some stay in secrets, and others moving to properties